### ISSUE TITLE
Bad text in popup for duplicate addr

### ISSUE THREAD

**Issue by frodrigo** [osm-fr/osmose-backend#84]
http://osmose.openstreetmap.fr/de/map/#zoom=18&lat=47.410722&lon=8.454623&layer=Mapnik&overlays=FFFFFFFFFFFFFFFFFFFFT&item=xxxx&level=1%2C2%2C3&tags=&fixable=

```
Hausnummer doppelt in der Straße
Multiple numbers "(<function <lambda> at 0x47ab578>, [u'36', None, None])" in way "Hönggerstrasse"
```

**Comment by jocelynj** [comment]
It must be because of this change in commit 6c06046745840505e38b7c92bbf296c9588b5e17:
```diff
-                "text":{"fr": u"Multiples numéros \"%s\" dans la voie \"%s\"" % (res[4], res[3]), "en": u"Multiple numbers \"%s\" in way \"%s\"" % (res[4], res[3])} } )
+                "text":{"fr": u"Multiples numéros \"%s\" dans la voie \"%s\"" % ((lambda z: z, res[4:]), res[3]), "en": u"Multiple numbers \"%s\" in way \"%s\"" % ((lambda z: z, res[4:]), res[3])} } )
```
Are you sure that we need lambda here ?

**Comment by frodrigo** [comment]
Missing a bit of code. Meaning is filter not null string and join it with ", ".

**Comment by jocelynj** [comment]
Should be fixed by 074ba42ad3b05101d9f363506689e5ccab50954d.

**Comment by jocelynj** [comment]
It seems to fail:
2015-12-27 19:49:53         "text": T_(u"Multiple numbers \"%(numbers)s\" in way \"%(way)s\"", {"numbers": ", ".join(lambda z: z, res[4:]), "way": res[3]}),
2015-12-27 19:49:53     TypeError: join() takes exactly one argument (2 given)

Shouldn't we replace .join(lambda z: z, res[4:]) by .join(res[4:]) ?

**Comment by jocelynj** [comment]
Fixed by ab16a7844ffac38087183b49456fba3b31a8caa5.
